apps/omg/lib/omg/state.ex

```diff
@@ -39,7 +38,6 @@ defmodule OMG.State do
     GenServer.start_link(__MODULE__, :ok, name: __MODULE__)
   end
 
-  @decorate measure_event()
```

Comment:
sometimes you replace these with `@decorate span` and sometimes you just remove, why?

Reply:
because they need to be replace - but not so easily. Appsignal traces don't work the same as spandex ones. So I thought I'd do them in a later PR.